app/controllers/krikri/validation_reports_controller.rb

```diff
@@ -0,0 +1,57 @@
+module Krikri
+  ##
+  # Marshals SolrDocuments for views.
+  # Sets default Solr request params for Validation Reports.
+  #
+  # ValidationReportsController inherits from the host application's
+  # ApplicationController.  It does not interit from Krikri's
+  # ApplicationController.
+  class ValidationReportsController < CatalogController
+    before_action :authenticate_user!
+    before_filter :valid_params, :only => :index
+
+    ##
+    # ValidationReportsController has access to views in the following
+    # directories:
+    #   krikri/validation_reports
+    #   catalog (defined in Blacklight)
+    # It inherits view templates from the host application's
+    # ApplicationController.  It uses krikri's application layout:
+    layout 'krikri/application'
```

Comment:
The rendered output looks like it's still calling `'layouts/blacklight'` per the error described [here](https://github.com/dpla/KriKri/pull/21#issuecomment-70331871).